src/control-plane/backend/lambda/api/common/utils.ts

```diff
@@ -835,6 +843,11 @@ function pathNodesToAttribute(nodes: IMetadataRawValue[] | undefined) {
   return pathNodes;
 }
 
+function getAppRegistryApplicationArn(pipeline: IPipeline): string {
+  return SERVICE_CATALOG_SUPPORTED_REGIONS.includes(pipeline.region) ?
```

Comment:
How about we always fetch the ARN of the appregsitry application? If it's not found, the value is empty.

Reply:
update the workflow to be consistent in all the regions